pip/req/req_set.py

```diff
@@ -307,12 +207,12 @@ def add_requirement(self, install_req, parent_req_name=None,
 
     def has_requirement(self, project_name):
         name = project_name.lower()
-        if (name in self.requirements and
-           not self.requirements[name].constraint or
-           name in self.requirement_aliases and
-           not self.requirements[self.requirement_aliases[name]].constraint):
-            return True
-        return False
+        return (
+            name in self.requirements and
+            not self.requirements[name].constraint or
+            name in self.requirement_aliases and
+            not self.requirements[self.requirement_aliases[name]].constraint
+        )
```

Comment:
This looks like it was just reformatting? If so can we just revert this back to the existing code. A separate PR could clean this up, but atm I think it's just adding noise to an already large diff.